tests/Form/TypeGuesser/CronExpressionTypeGuesserTest.php

```diff
@@ -9,14 +9,22 @@
 use Setono\CronExpressionBundle\Form\TypeGuesser\CronExpressionTypeGuesser;
 use Symfony\Component\Form\Guess\Guess;
 use Symfony\Component\Form\Guess\TypeGuess;
+use Symfony\Component\PropertyInfo\Extractor\PhpDocExtractor;
+use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+use Symfony\Component\PropertyInfo\PropertyInfoExtractor;
 
 final class CronExpressionTypeGuesserTest extends TestCase
 {
     private CronExpressionTypeGuesser $typeGuesser;
 
     public function setUp(): void
     {
-        $this->typeGuesser = new CronExpressionTypeGuesser();
+        $extractor = new PropertyInfoExtractor([], [
+            new PhpDocExtractor(),
+            new ReflectionExtractor(),
+        ]);
```

Comment:
Oh, this is what I mean by my other comment about initializing the property type extractor yourself.